metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -218,6 +218,12 @@ object ServerCommands {
     "[string], where the string is a directory location for the new file."
   )
 
+  val NewScalaProject = new Command(
+    "new-scala-project",
+    "New Scala Project",
+    """Create a new Scala project, works with sbt"""
```

Comment:
Should we make this more general instead of explicitly saying sbt? I know you mentioned that we should get Mill templates etc in the future.

Reply:
Ach, actually this will work also without sbt. That is a leftover from when I didn't know we could just used g8 explicitely. Will need to fix that! Thanks!